config/Makefile.mingw64

```diff
@@ -156,7 +156,7 @@ MKEXE_BOOT=$(CC) $(CFLAGS) $(LDFLAGS) $(OUTPUTEXE)$(1) $(2)
 MKEXE_ANSI=$(FLEXLINK) -exe
 
 ### How to build a static library
-MKLIB=rm -f $(1); $(TOOLPREF)ar rc $(1) $(2); $(RANLIB) $(1)
+MKLIB=rm -f $(1) && $(TOOLPREF)ar rcs $(1) $(2)
 #ml let mklib out files opts =
 #ml   Printf.sprintf "rm -f %s && %sar rcs %s %s %s"
```

Comment:
This `rm -rf` also wants to go (as in the .mingw Makefile)

Reply:
It is gone in the current version (commit 897c1f5, pushed yesterday).  Not sure why Github is making you review an outdated version!